tekton/src/tasks/gitops/gitops-suite-smtp-config-sendgrid.yml.j2

```diff
@@ -237,6 +243,7 @@ spec:
         --smtp-default-sender-name "${SMTP_DEFAULT_SENDER_NAME}" \
         --smtp-default-recipient-email "${SMTP_DEFAULT_RECIPIENT_EMAIL}" \
         --smtp-default-should-email-passwords "${SMTP_DEFAULT_SHOULD_EMAIL_PASSWORDS}" \
+        --smtp-disabled-templates "${SMTP_DISABLED_TEMPLATES}" \
```

Comment:
If SMTP_DISABLED_TEMPLATES is empty string then won't this break the parameter passing? i,.e. it would be 
```
--smtp-disabled-templates  --mas-smtpcfg-pod-template-yaml

```
we don't need to pass it as a paramter as it will get picked up as an env var

Reply:
It won't break as the argument is wrapped in quotes, it would be:
```
--smtp-disabled-templates  "" --mas-smtpcfg-pod-template-yaml
```

I've tested this task with no default templates specified and it works fine; see "disabled_templates unspecified, sendgrid" heading under https://github.ibm.com/maximoappsuite/saas-tekton/pull/101

In general, I'm passing in the env vars as params (even though not actually necessary) since I think it makes it the code more explicit and easier to understand,